src/main/utils/channel/channel-listeners/handling-of-channels.injectable.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { disposer, iter } from "../../../../common/utils";
+import type { RequestChannel } from "../../../../common/utils/channel/request-channel-listener-injection-token";
+import { getStartableStoppable } from "../../../../common/utils/get-startable-stoppable";
+import enlistRequestChannelListenerInjectable from "./enlist-request-channel-listener.injectable";
+import { requestChannelListenerInjectionToken } from "./listener-tokens";
+
+const listenerOfRequestChannelsInjectable = getInjectable({
+  id: "listener-of-request-channels",
+  instantiate: (di) => {
+    const enlistRequestChannelListener = di.inject(enlistRequestChannelListenerInjectable);
+    const requestChannelListeners = di.injectMany(requestChannelListenerInjectionToken);
```

Comment:
Note for conversation from elsewhere: this list of `requestChannelListeners` isn't reactive, and new listeners will not get "enlisted" if they pop into existence (ie. get registered) after `listenerOfRequestChannelsInjectable` has already `started` (as in `startableStoppable`). I'd say this is a bug waiting to happen.

How about a `startableStoppable` which reacts (eg. with MobX `autorun`, `reaction` or `observe`) to `computedInjectMany` of `requestChannelListenerInjectionToken` to coordinate enlisting and de-enlisting of listeners?

Reply:
I am going to do this in a separate PR because it is quite technical and can be done separately.